bootstrap-salt.sh

```diff
@@ -1793,7 +1793,7 @@ __apt_key_fetch() {
     url=$1
 
     # shellcheck disable=SC2086
-    apt-key adv ${_GPG_ARGS} --fetch-keys "$url"; return $?
+    curl ${_CURL_ARGS} -f -s -S -L "$url" | apt-key add - ; return $?
```

Comment:
You don't need to explicitly install and use `curl`, we have special wrapper function for downloading things: `__fetch_url` [here](https://github.com/saltstack/salt-bootstrap/blob/develop/bootstrap-salt.sh#L755).